content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,199 @@
+```
 
-If you do not use the Serverless Framework to deploy your project, you can use the following method to remove any unneeded files.
+Update the Serverless configuration file to package the `arm64` binary, as follows:
 
-1. Create a zip script that only zips the files that your project needs. [Example script](https://github.com/prisma/ecosystem-tests/blob/dev/platforms-serverless/lambda/zip.sh).
+```code file=serverless.yml
+package:
+  patterns:
+    - '!node_modules/.prisma/client/libquery_engine-*'
+    - 'node_modules/.prisma/client/libquery_engine-linux-arm64-*'
+    - '!node_modules/prisma/libquery_engine-*'
+    - '!node_modules/@prisma/engines/**'
+```
 
-2. Upload the zip. For example, to do this with the [AWS CLI](https://aws.amazon.com/cli/), you might use a command similar to the following:
+### Connection pooling
 
-   ```terminal wrap
-   aws lambda update-function-code --function-name my-function --zip-file "fileb://lambda.zip"
-   ```
+Generally, when you use a FaaS (Function as a Service) environment to interact with a database, every function invocation can result in a new connection to the database. This is not a problem with a constantly running node.js server. Therefore, it is beneficial to pool DB connections to get better performance. You can use the [Data Proxy](/data-platform/data-proxy) as a solution to this issue. For other solutions, see the [connection management guide for serverless environments](/guides/performance-and-optimization/connection-management#serverless-environments-faas).
 
 ## Summary
 
-Congratulations! You have successfully deployed the API to AWS Lambda.
-
-For more information about how to use the Serverless Framework with Prisma projects, see [use Prisma with the Serverless Framework](/guides/deployment/deployment-guides/use-prisma-with-serverless-framework).
-
-For more insight into the Prisma Client API, look at the function handlers in the `handlers/` folder.
-
-<Admonition type="info">
-
-**A note on connection pooling**<br /><br />Generally, when you use a FaaS (function as a service) environment to interact
-with a database, every function invocation can result in a new connection to the
-database. This is not a problem with a constantly running node.js server.
-Therefore, it is beneficial to pool DB connections to get better performance.
-You can use the [Data Proxy](/data-platform/data-proxy) as a solution to this issue.
-For other solutions, see the [connection management guide for
-serverless
-environments](/guides/performance-and-optimization/connection-management#serverless-environments-faas).
+Congratulations! You have successfully deployed your Serverless Framework application with Prisma!
 
-</Admonition>
+For more insight into the Prisma Client API, [check out our API reference](/reference/api-reference/prisma-client-reference) or connect with us on [GitHub](https://github.com/prisma/prisma), [Slack](https://slack.prisma.io), or [Discord](https://pris.ly/discord)
```

Comment:
A bit weird sentence. Do we put that on all guides?

Reply:
Happy to have feedback on it. Was thinking we have a block at the bottom of our guides saying where to learn more and how to get in contact with us.

Reply:
A bit unexpected to me why we have this on some pages, and not on all. 
Doesn't the docs footer also include links to GH, Slack and Discord?

But also not that important. If you like this, keep it.